contributors/devel/sig-api-machinery/generating-clientset.md

```diff
@@ -1,6 +1,6 @@
 # Generation and release cycle of clientset
 
-Client-gen is an automatic tool that generates [clientset](../design-proposals/api-machinery/client-package-structure.md#high-level-client-sets) based on API types. This doc introduces the use of client-gen, and the release cycle of the generated clientsets.
+Client-gen is an automatic tool that generates [clientset](../../design-proposals/api-machinery/client-package-structure.md#high-level-client-sets) based on API types. This doc introduces the use of client-gen, and the release cycle of the generated clientsets.
```

Comment:
Thanks for catching this! :)

Since relative urls can get stale quickly, can you change this to `./contributors/design-proposals/api-machinery/client-package-structure.md#high-level-client-sets`?

Reply:
done. @nikhita  PTAL